examples/yoti_example_flask/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3
+ARG YOTI_APPLICATION_ID
+ARG YOTI_CLIENT_SDK_ID
+ARG YOTI_FULL_KEY_FILE_PATH
+ADD . /code
+WORKDIR /code
+RUN pip install --no-cache-dir -r requirements.txt
+RUN sed -i "s|environ.get('YOTI_APPLICATION_ID')|'$YOTI_APPLICATION_ID' |g" settings.py
+RUN sed -i "s|environ.get('YOTI_CLIENT_SDK_ID')|'$YOTI_CLIENT_SDK_ID' |g" settings.py
+RUN sed -i "s|environ.get('YOTI_KEY_FILE_PATH')|'$YOTI_FULL_KEY_FILE_PATH' |g" settings.py
```

Comment:
Maybe you should check if these arguments have been set by the user, for example YOTI_APPLICATION_ID is an integer.